lib/capybara/screenshot/diff.rb

```diff
@@ -14,7 +14,7 @@ module Screenshot
     mattr_accessor :blur_active_element
     mattr_accessor :enabled
     mattr_accessor :hide_caret
-    mattr_reader(:root) { (defined?(Rails.root) && Rails.root) || "." }
+    mattr_reader(:root) { (defined?(Rails.root) && Rails.root) || Pathname(".") }
```

Comment:
it should use `self.root=` which will make it pathname, or I'm wrong?

Reply:
I just sets the value directly without using the setter method.  Apparently no test exercises this.

Reply:
yep, I tried to remove Rails.root stub, in order to verify this, but it's a little messy, so I dropped